scripts/export_precip.py

```diff
@@ -6,7 +6,11 @@
 
 
 def export_precip_2018():
-    data_path = Path('../data')
+    # if the working directory is alread ml_drought don't need ../data
+    if Path('.').absolute().as_posix().split('/')[-1] == 'ml_drought':
```

Comment:
allow to run from different locations (e.g. a notebook) so you need the path to be 'data' only

Sorry i accidentally pushed the same git comnmit message